docs/customize/users.md

```diff
@@ -0,0 +1,219 @@
+# Adding Users to Invenio ILS instance
+
+This guide provides step-by-step instructions for adding users to a custom Invenio App ILS instance using:
```

Comment:
I am not 100% sure that readers know what is `Invenio App ILS`, given that we refer to it as `InvenioILS`:
```suggestion
This guide provides step-by-step instructions for adding users to your own InvenioILS instance using two possible different methods:
```